vespajlib/src/main/java/com/yahoo/concurrent/lock/Locking.java

```diff
@@ -0,0 +1,30 @@
+package com.yahoo.concurrent.lock;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.locks.ReentrantLock;
+
+/**
+ * @author valerijf
+ */
+public class Locking {
```

Comment:
It's placed in concurrent/lock to be as available to others as Locks and Lock? There's nothing here that indicates this is used for locking in jdisc containers - shouldn't that at least be mentioned in case someone wants to change it? If this class stays here, then perhaps ClassLock, ClassMutex, or ClassLocking?

Reply:
Sounds good, what do you think @gjoranv ?